src/registrar/templates/django/admin/includes/contact_detail_table.html

```diff
@@ -0,0 +1,42 @@
+{% load i18n static %}
```

Comment:
Blocking: I do think these tables (and the ones for other contacts) require a thead, tr, th colspan=3, innerHTML of Contact details.

Another option would've been to mark this up as a definition list and tweak its CSS a bit in django-admin scss. I like the first option though.

```
<dl {% if field_name %} id="id_detail_table__{{field_name}}" {% endif %}>
    <dt>Title</dt>
    {% if user.title or user.contact.title %}
        {% if user.contact.title %}
        <dd>{{ user.contact.title }}</dd>
        {% else %}
        <dd>{{ user.title }}</dd>
        {% endif %}
    {% else %}
    <dd>Nothing found</dd>
    {% endif %}

    <dt>Email</dt>
    {% if user.email or user.contact.email %}
        {% if user.contact.email %}
        <dd>{{ user.contact.email }}</dd>
        {% else %}
        <dd>{{ user.email }}</dd>
        {% endif %}
    {% else %}
    <dd>Nothing found</dd>
    {% endif %}

    <dt>Phone</dt>
    {% if user.phone or user.contact.phone %}
        {% if user.contact.phone %}
        <dd>{{ user.contact.phone }}</dd>
        {% else %}
        <dd>{{ user.phone }}</dd>
        {% endif %}
    {% else %}
    <dd>Nothing found</dd>
    {% endif %}

</dl>
```